webpack/assets/javascripts/react_app/components/SearchBar/SearchBar.test.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import {
+  render,
+  fireEvent,
+  screen,
+  act,
+  waitFor,
+} from '@testing-library/react';
+import SearchBar from '.';
+import { Provider } from 'react-redux';
+import store from '../../redux';
+import {
+  SearchBarProps,
+  mockModelsEmptyAutocomplete,
+  mockModelsHardwareAutocomplete,
+  mockNotRecognizedResponse,
+} from './SearchBar.fixtures';
+
+jest.mock('../../redux/API/API', () => ({
+  get: async (url, header, params) => {
+    if (url === 'model/auto_complete_search') {
+      if (!params?.search) {
+        return { data: mockModelsEmptyAutocomplete };
+      }
+      if (params?.search === 'hardware_model  = ') {
+        return { data: mockModelsHardwareAutocomplete };
+      }
+      if (params?.search === 'wrong = nope') {
+        return { data: mockNotRecognizedResponse };
+      }
+    }
+    if (url === '/api/bookmarks?search=controller%3Dmodels&per_page=all') {
+      return {
+        data: {
+          total: 12,
+          subtotal: 1,
+          page: 1,
+          per_page: 12,
+          search: 'controller=models',
+          sort: {
+            by: null,
+            order: null,
+          },
+          results: [
+            {
+              name: 'name = a',
+              controller: 'models',
+              query: 'name = a',
+              public: true,
+              id: 7,
+              owner_id: 16,
+              owner_type: 'User',
+            },
+          ],
+        },
+      };
+    }
+  },
+}));
+
+describe('SearchBar', () => {
+  it('edit', async () => {
+    render(
+      <Provider store={store}>
```

Comment:
Is this the first React Testing Library test in Foreman?  Nice..

You may look at writing a `renderWithRedux` function like we have in Katello, so you don't have to render a provider in every test.